js/helper.js

```diff
@@ -348,6 +391,84 @@ function getProjectFlagsFromOptions({ projects = [] } = {}) {
   return projects.reduce((flags, project) => flags.concat('-p', project), []);
 }
 
+/**
+ * Determines an appropriate success message based on the command.
+ *
+ * @param {string[]} args Command line arguments passed to sentry-cli
+ * @returns {string|null} Success message to display or null if no message needed
```

Comment:
Maybe it is preferable to type this just as string, using empty string instead of null? Not sure how we typically do it in JS, so I'll defer to you on this.

```suggestion
 * @returns {string} Success message to display or empty string if no message needed
```